src/modules/displaymanager/displaymanager.conf

```diff
@@ -71,5 +71,6 @@ sysconfigSetup: false
 greetd:
   greeter_user: "tom_bombadil"
   greeter_group: "wheel"
+  greeter_css_location: "/etc/greetd/style.css"
```

Comment:
There should be some comments above that describe what this is and how to use it.